src/bin/generate_test_data.rs

```diff
@@ -23,10 +23,13 @@ async fn main() -> anyhow::Result<()> {
     }
 
     let nsn = NewSeason::new("Test NMG", "alttp", "Any% NMG", &mut db)?;
-    let sn = nsn.save(&mut db).unwrap();
+    let mut sn = nsn.save(&mut db).unwrap();
+    sn.set_state(SeasonState::QualifiersOpen, &mut db)?;
+    sn.set_state(SeasonState::QualifiersClosed, &mut db)?;
+    sn.set_state(SeasonState::Started, &mut db)?;
+    sn.update(&mut db)?;
```

Comment:
its silly that i dont have a way to move a Season through states that's smoother than this, but w/e, it's good enough for now